webplugin/scss/abstracts/_variables.scss

```
@@ -48,6 +48,8 @@ $rich-msg-width: 264px;
 // CARD IMAGE HEIGHT
 $card-img-height: 160px;
 
+$km-font-meta-contrast-color: #737373;
```

Stylelint (17.6.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)